pkg/pub_dartdoc/bin/pub_dartdoc.dart

```diff
@@ -6,20 +6,33 @@ import 'package:dartdoc/dartdoc.dart';
 import 'package:dartdoc/options.dart';
 
 import 'package:pub_dartdoc/pub_data_generator.dart';
+import 'package:pub_dartdoc/src/pub_hooks.dart';
 
 Future<void> main(List<String> arguments) async {
-  final config = await parseOptions(pubPackageMetaProvider, arguments);
+  // pub hooks
+  final pubResourceProvider =
+      PubResourceProvider(pubPackageMetaProvider.resourceProvider);
```

Comment:
Where is `pubPackageMetaProvider` coming from??

Reply:
nvm, https://pub.dev/documentation/dartdoc/latest/dartdoc/pubPackageMetaProvider.html

I thought it was missing :D